# This patch has been fed upstream.
# Patch to remove links to documents that are no longer available, fix
# changed links.
# http://bugzilla.cyrusimap.org/show_bug.cgi?id=3930

diff -rupN old/doc/index.html new/doc/index.html
--- old/doc/index.html	2016-04-07 17:43:16.583489776 -0700
+++ new/doc/index.html	2016-04-12 11:01:09.353415779 -0700
@@ -40,7 +40,6 @@ library distribution</B></A>
 <b>Special Platforms</b>
 <ul>
 <li> <a href="macosx.html"><b>Mac OS X Build Guide</b></a>
-<li> <a href="os390.html"><b>OS/390 Build Guide</b></a>
 <li> <a href="windows.html"><b>Win32 Build Guide</b></a>
 </ul>
 
diff -rupN old/doc/install.html new/doc/install.html
--- old/doc/install.html	2016-04-07 17:43:16.597328339 -0700
+++ new/doc/install.html	2016-04-12 11:01:33.989542591 -0700
@@ -218,7 +218,6 @@ can be linked against other dynamic obje
 library file extension is ".so", or where libtool creates the .la
 files correctly.  There is also documentation for
 <a href=windows.html>Win32</a>, <a href=macosx.html>MacOS X</a>, and
-<a href=os390.html>OS/390</a>.
 
 <hr>
 Back to the <a href="index.html">index</a>
diff -rupN old/doc/readme.html new/doc/readme.html
--- old/doc/readme.html	2016-04-07 17:43:16.589392684 -0700
+++ new/doc/readme.html	2016-04-12 11:02:38.062666985 -0700
@@ -102,7 +102,7 @@ we only have static Krb5 libraries; the
 these libraries in on platforms that support it (Solaris and Linux
 among them) but it does not.  It also doesn't always get the runpath
 of libraries correct.
-<li>Also see our <A HREF=http://bugzilla.andrew.cmu.edu>bugzilla</A>.
+<li>Also see our <A HREF="http://bugzilla.cyrusimap.org/index.cgi">bugzilla</A>.
 </ul>
 
 <H2>AUTHORS</H2>
